pytorch_lightning/trainer/progress.py

```diff
@@ -151,6 +151,9 @@ def from_defaults(cls, tracker_cls: Type[ReadyCompletedTracker], **kwargs: int)
     def reset_on_epoch(self) -> None:
         self.current.reset()
 
+    def reset_on_run(self) -> None:
```

Comment:
seems like deprecation public API, shall we make it back compatible?

Reply:
There's no need for this. Loops customization is not public yet